script.js

```diff
@@ -1,4 +1,123 @@
+var player1Score = 0;
+var player2Score = 0;
+//total score for player
+var totalPlayer1Score = [];
+var totalPlayer2Score = [];
+var totalSumPlayer1 = 0;
+var totalSumPlayer2 = 0;
+//dices for players
+var player1Dice1 = 0;
+var player1Dice2 = 0;
+var player2Dice1 = 0;
+var player2Dice2 = 0;
+
+var gameMode = `player one roll`;
+
 var main = function (input) {
```

Comment:
This is a really clean main and that's really good practice. When I say clean, I mean that you've abstracted away all the implementation details, such that you're basically just executing helper functions in main, making the main function easy to read